extensions/api/identity-api/verifiable-credentials-api/src/main/java/org/eclipse/edc/identityhub/api/verifiablecredentials/v1/unstable/VerifiableCredentialsApi.java

```diff
@@ -74,13 +67,10 @@ public interface VerifiableCredentialsApi {
                             content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiErrorDetail.class)), mediaType = "application/json"))
             }
     )
-    void addCredential(String participantId, VerifiableCredentialManifest manifest, SecurityContext securityContext);
+    void addCredential(String participantContextId, VerifiableCredentialManifest manifest, SecurityContext securityContext);
 
     @Operation(description = "Update an existing VerifiableCredential.",
             operationId = "updateCredential",
-            parameters = {
```

Comment:
why is this removed? (question applies to all controllers in this PR)

Reply:
This creates the path param twice in the swagger spec, making it invalid. 
Also it is not possible in the java API to define a path param at the API level, so it should be declared for every method, even it not used.